R/run_ml.R

```diff
@@ -144,6 +146,7 @@ run_ml <-
            group_partitions = NULL,
            corr_thresh = 1,
            seed = NA,
+           impute_in_training = FALSE,
```

Comment:
I would name this parameter `impute_after_split` to make it clear that the imputation occurs after the train/test split and before model training and testing.